src/js/components/cycleCountReporting/IndicatorsTab/IndicatorNumberCards.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiInformationLine } from 'react-icons/ri';
+import { SortableContainer } from 'react-sortable-hoc';
+
+import LoadingNumbers from 'components/dashboard/LoadingNumbers';
+import NumberCard from 'components/dashboard/NumberCard';
+
+const SortableIndicatorCards = SortableContainer(({ data }) => (
+  <div className="card-component">
+    {data.map((card, index) => (
+      <NumberCard
+        index={index}
+        cardTitle={card.title}
+        cardTitleDefaultValue={card.titleDefaultValue}
+        cardNumberType={card.numberType}
+        cardNumber={card.number}
+        cardType={card.type}
+        cardSubtitle={card.subtitle}
+        cardSubtitleDefaultValue={card.subtitleDefaultValue}
+        cardSubtitleValue={card.subtitleValue}
+        cardInfo={card.info}
+        cardInfoDefaultValue={card.infoDefaultValue}
+        showPercentSign={card.showPercentSign}
+        infoIcon={<RiInformationLine size={20} />}
+        disabled
+        hideDraghandle
+      />
+    ))}
+  </div>
+));
+
+const IndicatorNumberCards = ({
+  loading,
+  numberCards,
```

Comment:
what is `numberCards`? It might be difficult to understand the context without digging deeper into the code. Maybe a different name fits better?